src/components/Dialog-Window/Audio/DialogAudio.module.scss

```
@@ -0,0 +1,14 @@
+.description {
+  font-family: Roboto;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 1rem;
+  line-height: 0rem;
+  margin-top: 8px;
```

Margins in and around text should use `rem`, to provide good readability with any font size.

```suggestion
  margin-top: 0.5rem;
```